extra_model/_cli.py

```diff
@@ -6,28 +6,50 @@
 
 from extra_model._errors import ExtraModelError
 from extra_model._run import run
+from extra_model._setup import setup
 
 logger = logging.getLogger(__name__)
 
 
-DEFAULT_OUTPUT_PATH = "/io"
+OUTPUT_PATH = "/io"
+EMBEDDINGS_PATH = "/embeddings"
 
 
 @click.command()
 @click.argument("input_path", type=Path)
-@click.argument("output_path", type=Path, default=DEFAULT_OUTPUT_PATH)
+@click.argument("output_path", type=Path, default=OUTPUT_PATH)
 @click.option("--debug", is_flag=True, help="Enable debug logging")
 def entrypoint(input_path: Path, output_path: Path, debug: bool = False) -> None:
     """Run the Extra algorithm for unsupervised topic extraction.
 
     INPUT_PATH is the path to the input parquet file with the user generated texts.
     
-    OUTPUT_PATH is the path to the output directory. Default is `/io`
+    OUTPUT_PATH is the path to the output directory. Default is `/io`.
     """
     logging.getLogger("extra_model").setLevel("DEBUG" if debug else "INFO")
 
     try:
         run(input_path, output_path)
+        sys.exit(0)
+
     except ExtraModelError as e:
         logger.exception(e) if debug else logger.error(e)
         sys.exit(1)
+
+
+@click.command()
+@click.argument("output_path", type=Path, default=EMBEDDINGS_PATH)
+def entrypoint_setup(output_path: Path) -> None:
```

Comment:
Can you update the "Quickstart" with the command that one should run to get the embeddings?

Reply:
sounds good!